ellama.el

```diff
@@ -179,8 +183,16 @@ in.  Default value is (current-buffer).
 :point POINT -- POINT is the point in buffer to insert ellama reaply at."
   (let* ((buffer (or (plist-get args :buffer) (current-buffer)))
 	 (point (or (plist-get args :point)
-		    (with-current-buffer buffer (point)))))
+		    (with-current-buffer buffer (point))))
+	 (filter (or (plist-get args :filter) #'identity))
```

Comment:
All new keys should be added to docstring

Reply:
Done!